collectors/ebpf.plugin/ebpf_mount.c

```diff
@@ -386,7 +386,7 @@ static void mount_collector(ebpf_module_t *em)
 static void ebpf_create_mount_charts(int update_every)
 {
     ebpf_create_chart(NETDATA_EBPF_MOUNT_GLOBAL_FAMILY, NETDATA_EBPF_MOUNT_CALLS,
-                      "Calls to mount and umount syscalls.",
+                      "Calls to syscalls",
```

Comment:
Do you really want to make titles look like units?

Reply:
We want to simplify them. I returned to previous version here.